lib/Cmts.ml

```diff
@@ -137,7 +136,7 @@ end = struct
               , next.loc_start.pos_lnum - loc.loc_end.pos_lnum )
             with
             | 0, 0 -> impossible "already tested by previous if"
-            | 0, _ when infix_symbol_before src prev -> `Before_next
+            | 0, _ when infix_symbol_before src loc -> `Before_next
```

Comment:
What is this diff about ?

Reply:
the former implementation was using the location of the `prev` element (before the comment) and we then looked for the position of the last token at this position; and now we look for the token preceding the location `loc` (which is the location of the comment)